web/gds-admin-ui/src/redux/auth/actions.js

```diff
@@ -0,0 +1,19 @@
+import { AuthActionTypes } from './constants';
+
+export const loginUser = (data) => ({
+    type: AuthActionTypes.LOGIN_USER,
+    payload: data
+});
```

Comment:
Is the plan to have this action eventually call the API server with the credentials and then dispatch, or is the dispatch going to happen from the Login component?

Reply:
The dispatch happen from the Login Component and I decode the passed credentials in the saga file

Reply:
To thus dispatch the user info in the Store

Reply:
Right, but eventually you'll have to make a request to the server to get the GDS Admin API token; I'm still working on that, but the Google Credential isn't the end, you'll be holding a GDS access and refresh token as the final say for if the login was successful. 

There is a way to configure Google to make the POST request if you'd like, but I don't think that makes sense for a React app.